### ISSUE TITLE
show_lines has no color by default

### ISSUE THREAD

**Issue by keflavich** [aplpy/aplpy#169]
Noted in https://github.com/aplpy/aplpy-examples/pull/1, apparently `show_lines` defaults to _no_ color.  There is no obvious reason for this that I can see in the source for `show_lines` or `LineCollection`:

```
F.show_grayscale()
F.show_lines([np.array([[265,267],[-28,-30]])]) # shows nothing
F.show_lines([np.array([[265,267],[-28,-30]])],color='r')
```

**Comment by gully** [comment]
I love open source.  I was at wit's end to help a graduate student who asked me why `show_lines(` wasn't working, I typed that substring into GitHub search and voilà: answer is here.
So I guess thank you all for contributing to open source and making this dialog public to help others.  Although the default of no color seems like a bug, right?

**Comment by keflavich** [comment]
@gully yes, I think this is fair to regard as a bug.  The user should either be warned that no color is selected or there should be a default color.  A pull request fixing the issue would be welcome.
